tutorials/basic-tutorial.md

```diff
@@ -17,16 +17,16 @@ The easiest way to try out the Library is to download the [kickstart.zip](https:
 ----------------
 
 > NOTE: To run all code examples from this tutorial you will need only Openscad and
-> a single file: constructive-compiled.scad put in the same Fodler as your own .scad files.
+> a single file: constructive-compiled.scad put it into OpenScad Library folder (File -> Show Library Folder...) OR at the same Folder as your own .scad files.
```

Comment:
As a developer you may want to have it locally, but I think for users it is more convenient to once place it into Library folder and have access to it from every project they may have.

But, I can change text if you want to. What do you prefer to have here?

Reply:
I would say having a lib in a single file in the Project Directory, keeps your model working and repeatable even if the lib breaks the compatibility. For actual development i use 
include constructive-all,scad instead of include constructive-compiled.scad. The former is working directly from sources not compiled into a single file. this one IMHO could actually go into lib folder, but it is not covered y the tutorial anyway.

So i would say in this case to leave it as it was (keeping the tutorial simple for beginners).
this does not mean i do no appreaciate the work you have done, it is just a different opinion.

Reply:
super!